arango_datasets/datasets.py

```diff
@@ -89,21 +92,45 @@ def load_file(self, collection_name: str, edge_type: bool, file_url: str) -> Non
             )
             print(exec.error_message)
             sys.exit(1)
-
-        try:
-            with progress(f"Downloading file for: {collection_name}") as p:
-                p.add_task("load_file")
-                data = requests.get(file_url).json()
-        except (HTTPError, ConnectionError) as e:
-            print("Unable to download file.")
-            print(e)
-            raise
-        print(f"Downloaded file for: {collection_name}, now importing... ")
-        self.insert_docs(collection, data, collection_name)
+        if self.file_type == "json":
+            try:
+                with progress(f"Downloading file for: {collection_name}") as p:
+                    p.add_task("load_file")
+                    data = requests.get(file_url).json()
+            except (HTTPError, ConnectionError) as e:
+                print("Unable to download file.")
+                print(e)
+                raise
+            print(f"Downloaded file for: {collection_name}, now importing... ")
+            self.insert_docs(collection, data, collection_name)
+
+        elif self.file_type == "jsonl":
+            json_data = []
+            try:
+                with progress(f"Downloading file for: {collection_name}") as p:
+                    p.add_task("load_file")
+                    data = requests.get(file_url)
+
+                if data.encoding is None:
+                    data.encoding = "utf-8"
+
+                for line in data.iter_lines(decode_unicode=True):
+                    if line:
+                        json_data.append(json.loads(line))
+
+            except (HTTPError, ConnectionError) as e:
+                print("Unable to download file.")
+                print(e)
+                raise
+            print(f"Downloaded file for: {collection_name}, now importing... ")
+            self.insert_docs(collection, json_data, collection_name)
```

Comment:
Could raise here for unsupported file types?
```suggestion
            self.insert_docs(collection, json_data, collection_name)
        else:
            raise ValueError(f"Unsupported file type: {self.file_type}")
```